src/shogun/machine/Machine.h

```diff
@@ -333,6 +333,21 @@ class CMachine : public CStoppableSGObject
 			return false;
 		}
 
+		/** Continue Training
+		 *
+		 * This is overloaded in IterativeLinearMachine
+		 * to resume training of a model that was prematurely stopped.
+		 *
+		 * NOT IMPLEMENTED!
+		 *
+		 * @return whether training was successful
+		 */
+		virtual bool continue_train()
+		{
+			SG_ERROR("Training continuation not supported by this model.\n");
```

Comment:
Should we have our own exception type for such things?
NotImplementedException
@vigsterkr 
SG_NOTIMPLEMENTED is more for "sorry not YET implemented, i.e. incomplete implementation)", as opposed to. "This machine doesnt support this feature"

Reply:
we can leave it as is for now

Reply:
or @shubham808 you can have a look at what is the apprpriate exception
@vinx13 could you explain how to use custom ones?

Reply:
you can use `SG_THROW(exception_type, message...)`, but that's in `feature/transformers` branch that haven't been merged